.github/workflows/build.yml

```diff
@@ -38,6 +38,9 @@ jobs:
       - name: Build
         id: make_build
         run: |
+          export PATH="/usr/local/opt/llvm/bin:$PATH"
```

Comment:
Why don't you put them in Makefile?

Reply:
It makes more sense to me to set environment variables in here rather than in makefile.

Reply:
Hm.... if they are kinda "standard" ones in mac environment, just use the following expression in Makefile may be better?

LDFLAGS ?= "-L/usr/local/opt/llvm/lib"
CPPFLAGS ?= "...."

Reply:
I don't they they are "standard" ones, but I can be wrong. I think it would make more sense to not have them in Makefile if they are needed outside of the makefile process in the future. If not then they can be set as Makefile variables. What do you think?